.github/workflows/pr-ci.yml

```diff
@@ -1,85 +1,158 @@
       with:
         name: cryostat-${{ matrix.arch }}
-      if: github.repository_owner == 'cryostatio'
     - name: Load cryostat image
       run: podman load -i cryostat-${{ matrix.arch }}.tar
-      if: github.repository_owner == 'cryostatio'
     - name: Tag cryostat image
-      run: podman tag cryostat ghcr.io/${{ github.repository_owner }}/cryostat:pr-${{ github.event.number }}-${{ github.event.pull_request.head.sha }}-linux-${{ matrix.arch }}
-      if: github.repository_owner == 'cryostatio'
+      run: podman tag cryostat ghcr.io/${{ needs.checkout-branch.outputs.PR_repo_owner }}/cryostat:pr-${{ github.event.number }}-${{ needs.checkout-branch.outputs.PR_head_sha }}-linux-${{ matrix.arch }}
     - name: Push PR test image to ghcr.io
       id: push-to-ghcr
       uses: redhat-actions/push-to-registry@v2
       with:
         image: cryostat
-        tags: pr-${{ github.event.number }}-${{ github.event.pull_request.head.sha }}-linux-${{ matrix.arch }}
-        registry: ghcr.io/${{ github.repository_owner }}
-        username: ${{ github.event.pull_request.user.login }}
-        password: ${{ secrets.GHCR_PR_TOKEN }}
-      if: github.repository_owner == 'cryostatio'
-    - name: Comment test image link
-      uses: thollander/actions-comment-pull-request@v1
+        tags: pr-${{ github.event.number }}-${{ needs.checkout-branch.outputs.PR_head_sha }}-linux-${{ matrix.arch }}
+        registry: ghcr.io/${{ needs.checkout-branch.outputs.PR_repo_owner }}
+        username: ${{ github.event.comment.user.login }}
+        password: ${{ secrets.GITHUB_TOKEN}}
+    - name: Store images as output amd64
+      if: ${{ matrix.arch }} == 'amd64'
+      id: amd64_image
+      run: echo "image=${{ steps.push-to-ghcr.outputs.registry-path }}" >> "$GITHUB_OUTPUT"
+    - name: Store images as output
+      if: ${{ matrix.arch }} == 'arm64'
+      id: arm64_image
+      run: echo "image=${{ steps.push-to-ghcr.outputs.registry-path }}" >> "$GITHUB_OUTPUT"
+
+  comment-image:
+    runs-on: ubuntu-latest
+    needs: [push-to-ghcr]
+    if: ithub.repository_owner == 'cryostatio' && github.event.issue.pull_request && startsWith(github.event.comment.body, '/build_test')
+    steps:
+    - name: Create markdown table
+      uses: petems/csv-to-md-table-action@master
+      id: md-table
+      env:
+        amd64_image: ${{ needs.push-to-ghcr.outputs.amd64_image }}
+        arm64_image: ${{ needs.push-to-ghcr.outputs.arm64_image }}
+      with: 
+        csvinput: |
+          ARCH, IMAGE
+          AMD64, ${{ env.amd64_image }}
+          ARM64, ${{ env.arm64_image }}
+    - uses: thollander/actions-comment-pull-request@v1
       with:
         message: |-
-          Test image available:
+          ${{ steps.md-table.outputs.markdown-table }}
+
+          To run smoketest:
           ```
-          $ CRYOSTAT_IMAGE=${{ steps.push-to-ghcr.outputs.registry-path }} sh smoketest.sh
+          CRYOSTAT_IMAGE=${PULL_REQUEST_IMAGE} sh smoketest.sh
```

Comment:
How about? This way, it is more convenient to copy and avoid confusion about the `PULL_REREQUEST_IMAGE` variable.

```
CRYOSTAT_IMAGE=${{ env.amd64_image }} sh smoketest.sh
```

Reply:
should I include a line for the arm64 image too?

Reply:
Good question. I think we just need one example.